src/main/proto/pb.proto

```diff
@@ -1412,6 +1417,13 @@ message MyVote {
     string reveal_tx_id = 5;
 }
 
+message ParamChangeEvent {
+    // We use odingal instead of Enum as enaums in PB are a pain... as well we want to be more flexible at changes
```

Comment:
The spelling in this comment could use some love :)

Reply:
ups... ;-) was late...

Reply:
I assume rest is utACK...

Reply:
Yes, utACK, forgot to add that.